### PR TITLE
fix(prometheus): Add support for bearer token to prometheus input plugin

### LINKED ISSUE #864: [Prometheus] Add token support
We just have to create a PR using the @jchauncey's commit:
https://github.com/jchauncey/telegraf/commit/58696bb4f3026b1b095f6bbfa91b13084c99b901

### PR DESCRIPTION
**Pull request by jchauncey**
closes #864

### CODE REVIEW

File: plugins/inputs/prometheus/prometheus.go [hunk 1]
```diff
@@ -59,9 +71,34 @@ var client = &http.Client{
...
+	var resp *http.Response
+
+	var rt http.RoundTripper = &http.Transport{
+		Dial: (&net.Dialer{
+			Timeout:   10 * time.Second,
+			KeepAlive: 30 * time.Second,
+		}).Dial,
+		TLSHandshakeTimeout: 10 * time.Second,
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true,
```
Comment (sparrc) [inline plugins/inputs/prometheus/prometheus.go]: should this line be `InsecureSkipVerify: p.InsecureSkipVerify,`?

Comment (jchauncey) [inline plugins/inputs/prometheus/prometheus.go]: doh sorry bout that. Yeah fixed it =)

**Comment by sparrc** [comment]
looks great, thanks @jchauncey, can you update CHANGELOG? I'll merge when the tests pass

**Comment by jchauncey** [comment]
Alright changelog has been updated. 

=)

**Comment by sparrc** [comment]
closed by https://github.com/influxdata/telegraf/commit/d6b5f3efe63cd41df23cebe245212bf7877280e9

**Pull request closed without merging**